test/test_jit.py

```diff
@@ -4701,6 +4701,10 @@ def consec(size, start=0):
             numel = torch.tensor(size).prod().item()
             return torch.arange(numel).view(size)
 
+        # copied from @agolynski
```

Comment:
you could maybe remove the `copied from ...` line, not sure that it really helps the test much

Reply:
haha ok. i thought we needed to let people know that this is a copied code